app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -200,20 +208,39 @@ public void askReasonAndExecute(Media media,
             String finalReason = reason;
 
             Single.defer((Callable<SingleSource<Boolean>>) () ->
-                    makeDeletion(context, media, finalReason))
```

Comment:
If I understand correctly, only the indentation was changed for this block. If you actually changed code, please let me know.
That makes the code coverage check ignorable.